packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -219,10 +219,10 @@ export class Combobox
 
   /**
    * Specifies the selection mode:
-   * - "multiple" allows any number of selected items (default),
-   * - "single" allows only one selection,
-   * - "single-persist" allow and require one open item,
-   * - "ancestors" is like multiple, but shows ancestors of selected items as selected, with only deepest children shown in chips.
+   * - `"multiple"` allows any number of selected items,
+   * - `"single"` allows only one selection,
+   * - `"single-persist"` allow and require one open item,
+   * - `"ancestors"` is like multiple, but shows ancestors of selected items as selected, with only deepest children shown in chips.
```

Comment:
Similar to the other instance _believe_ in this case both the `:` and `-` / `,` and `-` would be shown together. Maybe remove the `-` throughout so only one character is displayed in the doc? For instance, a screenshot of the [combobox page](https://developers.arcgis.com/calcite-design-system/components/combobox/):

![image](https://github.com/Esri/calcite-design-system/assets/5023024/570a3f29-f205-4406-9b80-d6d3a4456d13)